apps/api/prisma/schema.prisma

```diff
@@ -318,7 +320,9 @@ model Document {
   createdAt            DateTime @default(now())
   fileSize             Int?     @default(0)
   fileType             String?
+  representationId     Int?
 
+  representations  Representation?   @relation(fields: [representationId], references: [id])
```

Comment:
Can we confirm this link between user and representation.

Reply:
This is the representation to document one-to-many

Reply:
There is a link to User, will show you.